cmp/grid/GridModel.js

```diff
@@ -771,8 +770,12 @@ export class GridModel {
     async autosizeAsync(options = {}) {
         options = {...this.autosizeOptions, ...options};
 
-        const {columns} = options;
+        if (options.mode === GridAutosizeMode.DISABLED) {
```

Comment:
Should we change this to `!this.autosizeEnabled`?

Reply:
thought about that for way too long -- was thinking that if you somehow passed in an option  set (perhaps generated by a testing program) that had a mode: 'disabled' that it should also work.

Reply:
Ah yes - makes perfect sense